docs/upgrade-notes.asciidoc

```diff
@@ -179,4 +179,28 @@ It will no longer be possible to create new scripted fields directly from the *D
 
 *Action* +
 Migrate to runtime fields or ES|QL instead of creating new scripted fields. Existing scripted fields can still be edited or deleted.
+====
+
+
+[float]
+=== Known issue
+
+[discrete]
+[[known-issue-204384]]
```

Comment:
@lcawl @rudolf 
Is this ok for https://github.com/elastic/kibana/pull/204384#issuecomment-2550926654?